packages/mobile/e2e/src/usecases/NewAccountPhoneVerification.js

```diff
@@ -0,0 +1,252 @@
+    await dismissBanners()
+
+    // Proceed through education screens
+    for (let i = 0; i < 3; i++) {
+      await element(by.id('Education/progressButton')).tap()
+    }
+
+    // Create new account
+    await element(by.id('CreateAccountButton')).tap()
+
+    // Accept terms
+    await element(by.id('scrollView')).scrollTo('bottom')
+    await expect(element(by.id('AcceptTermsButton'))).toBeVisible()
+    await element(by.id('AcceptTermsButton')).tap()
+
+    // Set name
+    await element(by.id('NameEntry')).replaceText(EXAMPLE_NAME)
+    await element(by.id('NameAndPictureContinueButton')).tap()
+
+    // Set and verify pin
+    await enterPinUi()
+    await enterPinUi()
+
+    // Set phone number
+    await expect(element(by.id('PhoneNumberField'))).toBeVisible()
+    await element(by.id('PhoneNumberField')).replaceText(examplePhoneNumber)
+    await element(by.id('PhoneNumberField')).tapReturnKey()
+  })
+
+  // Uninstall after to remove verification
+  afterAll(async () => {
+    device.uninstallApp()
+  })
+
+  // Check Twilio balance at start
+  beforeAll(async () => {
+    try {
+      await checkBalance()
+    } catch {}
+  })
+
+  // Check that Twilio SID, Auth Token and Verification Phone Number are defined
+  // Check that Twilio balance is enought to complete tests
+  if (
+    TWILIO_ACCOUNT_SID &&
+    TWILIO_AUTH_TOKEN &&
+    VERIFICATION_PHONE_NUMBER &&
+    (async () => await checkBalance())
+  ) {
+    // Check status of 'https://staging-komenci.azurefd.net/v1/ready' prior to tests
+    // If Komenci is up run phone verification
+    // Else Verify that app handles Komenci being down appropriately
+    if (async () => await checkKomenci()) {
+      jest.retryTimes(2)
+      it('Then should be able to verify phone number', async () => {
+        // Get Date at start
+        let date = new Date()
+        // Start verification
+        await element(by.text('Start')).tap()
+
+        // Retrieve the verification codes from Twilio
+        const codes = await receiveSms(date)
+
+        // Wait for code input - 45 seconds max after we've received the last code
+        await waitFor(element(by.id('VerificationCode0')))
+          .toExist()
+          .withTimeout(45 * 1000)
+
+        // Check that we've received 3 codes
+        console.log(codes)
+        jestExpect(codes).toHaveLength(3)
+
+        // Enter 3 codes
+        for (let i = 0; i < 3; i++) {
+          await sleep(1000)
+          await waitFor(element(by.id(`VerificationCode${i}`)))
+            .toBeVisible()
+            .withTimeout(15 * 1000)
+          await element(by.id(`VerificationCode${i}`)).typeText(codes[i])
+        }
+
+        // Assert we've arrived at the home screen
+        await waitFor(element(by.id('SendOrRequestBar')))
+          .toBeVisible()
+          .withTimeout(45 * 1000)
+
+        // Assert that phone verification CTA is NOT served
+        try {
+          await element(by.id('CTA/ScrollContainer')).scroll(500, 'right')
```

Comment:
This could become a false positive, if we ever change the order of the notifications this will still pass even if the confirmation card is present. I think a better place to check would be the `Confirm Number` in the settings menu.

Reply:
Adjusted this check to use the presence or absence of 'Connect phone number' in settings as recommended.